hybrasyl/Scripting/Script.cs

```diff
@@ -103,14 +105,16 @@ public dynamic GetObjectWrapper(WorldObject obj)
         /// </summary>
         public void PrepareState()
         {
-            State.LoadCLRPackage();
-            State.DoString(@"Scripting = CLRPackage('Hybrasyl', 'Hybrasyl.Scripting')");
-            State.DoString(@"Enums = CLRPackage('Hybrasyl', 'Hybrasyl.Enums')");
-            State["world"] = Processor.World;
-            State["logger"] = new ScriptLogger(Name);
+
+            // Create index of functions
+         //   State.LoadCLRPackage();
+           // State.DoString(@"Scripting = CLRPackage('Hybrasyl', 'Hybrasyl.Scripting')");
+            //State.DoString(@"Enums = CLRPackage('Hybrasyl', 'Hybrasyl.Enums')");
+            //State["world"] = Processor.World;
+            //State["logger"] = new ScriptLogger(Name);
             // Prohibit future imports from other .NET assemblies
-            State.DoString(@"import = function() end");
-            State.DoString(@"CLRPackage = function() end");
+            //State.DoString(@"import = function() end");
+            //State.DoString(@"CLRPackage = function() end");
```

Comment:
more dead?

Reply:
I was leaving this with the expectation of revisiting but yes. dead